src/common/grpc.rs

```diff
@@ -883,16 +883,20 @@ where
     R: GrpcResponse + 'static,
 {
     fn unary_rpc(&mut self, method: &str, data: &Bytes) -> Result<Bytes, GrpcError> {
-        {
-            let cap = RefCell::borrow(&self.buffer).capacity();
-            let len = RefCell::borrow(&self.buffer).len();
-            log::debug!("current status of buffer is cap: {:?} len: {:?}", cap, len);
-        }
-        debug!("webRTC");
+        // {
```

Comment:
These commented out debug logs seem easy enough to recreate if needed again, maybe it is better to just drop them rather than leave them in commented out where they have a good chance of going stale.